renderdoc/driver/metal/metal_serialise.cpp

```diff
@@ -353,6 +378,43 @@ void DoSerialise(SerialiserType &ser, RDMTL::RenderPassDescriptor &el)
   SERIALISE_MEMBER(sampleBufferAttachments);
 };
 
+template <typename SerialiserType>
+void DoSerialise(SerialiserType &ser, RDMTL::ComputePassSampleBufferAttachmentDescriptor &el)
+{
+  // TODO: when WrappedCounterSampleBuffer exists
+  // SERIALISE_MEMBER(sampleBuffer);
+  SERIALISE_MEMBER(startOfEncoderSampleIndex);
+  SERIALISE_MEMBER(endOfEncoderSampleIndex);
+}
+
+template <typename SerialiserType>
+void DoSerialise(SerialiserType &ser, RDMTL::ComputePipelineDescriptor &el)
+{
+  SERIALISE_MEMBER(label);
+  SERIALISE_MEMBER(computeFunction);
+  SERIALISE_MEMBER(threadGroupSizeIsMultipleOfThreadExecution);
+  SERIALISE_MEMBER(maxTotalThreadsPerThreadgroup);
+  SERIALISE_MEMBER(maxCallStackDepth);
+  SERIALISE_MEMBER(stageInputDescriptor);
+  SERIALISE_MEMBER(buffers);
+  SERIALISE_MEMBER(supportIndirectCommandBuffers);
+  // TODO: when WrappedMTLDynamicLibrary exists
+  // SERIALISE_MEMBER(preloadedLibraries);
+  // Deprecated
```

Comment:
No need to change anything but a comment for the future : don't need the deprecated here as this method is about serialising the structure and not about what the data means. Also it might be misunderstood if the field is deprecated in the RDMTL structure as opposed to deprecated in the ObjC structure.